app/presenters/rummager_document_presenter.rb

```diff
@@ -10,6 +10,7 @@ class RummagerDocumentPresenter < ActionView::Base
   attr_reader :title, :link, :summary, :content_id
 
   def initialize(document_hash)
+    super()
```

Comment:
Yup, super is a special method, without parenthesis it will automatically pass in the methods arguments https://dannyh79.github.io/posts/ruby-super-super